config/resources/osdf.yaml

```diff
@@ -27,8 +27,3 @@ Federation:
 Registry:
   RequireCacheApproval: true
   RequireOriginApproval: true
-Director:
-  X509ClientAuthenticationPrefixes:
-  - /xenon/PROTECTED/
-  - /user/ligo/
-  - /igwn/
```

Comment:
How can I test that we don't break access for these namespaces? Or is the point that we're intentionally shutting things off for them? Do they know about it?